src/sentry/integrations/discord/utils/channel.py

```diff
@@ -1,20 +1,25 @@
 from __future__ import annotations
 
 from django.core.exceptions import ValidationError
+from requests.exceptions import Timeout
 
 from sentry.integrations.discord.client import DiscordClient
-from sentry.shared_integrations.exceptions import IntegrationError
+from sentry.shared_integrations.exceptions import ApiTimeoutError, IntegrationError
 from sentry.shared_integrations.exceptions.base import ApiError
 
 from . import logger
 
+# Same default timeout as Slack
+DISCORD_DEFAULT_TIMEOUT = 10
```

Comment:
We don't need this and line 22 anymore right?

Reply:
Good catch!